src/kakarot/gas.cairo

```diff
@@ -73,15 +73,18 @@ namespace Gas {
         return memory_cost;
     }
 
-    // @notice Compute the expansion cost of max_offset for the memory
+    // @notice Compute the expansion cost of max_offset for the memory.
+    // @dev Assumption max_offset < 2**133 necessary for unsigned_div_rem usage.
```

Comment:
should this assumption be verified? How are we sure that this case never happens during execution?